enzy_htp/_interface/amber_interface.py

```diff
@@ -2834,6 +2834,57 @@ def remove_traj_solvent(
         contents = "\n".join(contents)
         self.run_cpptraj(contents)
 
+    def get_rmsd(
+            self,
+            stru_esm: StructureEnsemble,
+            stru_selection: StruSelection,
+        ) -> float:
+        """Calculate the RMSD value of a StructureEnsemble instance.
+        
+        Args:
+            stru_esm (StructureEnsemble): A collection of different geometries of the same enzyme structure.
+            stru_selection (StruSelection): A StruSelection instance representing the region for calculating RMSD value.
+            
+        Returns:
+            rmsd_value (float): The average RMSD value of all the frames comparing with the average structure.    
+        """
+        tmp_dir = eh_config['system.SCRATCH_DIR']
+        tmp_nc_path=os.path.join(tmp_dir, "tmp_amber_traj.nc")
```

Comment:
Use fs.get_valid_temp_name to avoid overwritting

Reply:
Done.